source/security/auth-mechanisms/scram.txt

```diff
@@ -23,9 +23,8 @@ Overview
 **Salted Challenge Response Authentication Mechanism (SCRAM)** is a family of
 authentication mechanisms that use a challenge-response mechanism to authenticate
 the user. SCRAM-SHA-256, which uses the SHA-256 algorithm to hash your password, is the
-default authentication mechanism in {+mdb-server+} version 4.0
-and later. SCRAM-SHA-1, which uses the SHA-1 algorithm instead, is the default
-authentication mechanism in {+mdb-server+} versions earlier than 4.0.
+default authentication mechanism in {+mdb-server+} version 6.0
+and later.
```

Comment:
S: I think you can just say "the default authentication mechanism" or "the default authentication mechanism for all supported server versions"
```suggestion
default authentication mechanism.
```